### PR TITLE
[1]Specify usage

### PR DESCRIPTION
**Pull request by ufoo68**
# issue

#1 

# changes

- Describe example for use query on README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,42 @@
 # TheSportsDB
 GraphQL wrapper for TheSportsDB
 API doc - https://www.thesportsdb.com/api.php
+
+# Example of query
+
+Query for get types of schema
```
Comment (sefininio) [inline README.md]: Hi @ufoo68 ,
Could you please elaborate on why this documentation is needed in the README?
The beauty of GraphQL is that you don't have to document the API, it documents itself with the schema and types - all you have to do is open up the playground and everything is already documented and available.

I see value, however, in adding documentation on server execution details and how to access the playground. Would you like to edit the PR to include this information?

**Review by sefininio — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ufoo68) [reply README.md]: I understand. I will change the document describe how to execute server and access playground.

Comment (ufoo68) [reply README.md]: [f89e35c](https://github.com/jsBlackBelt/TheSportsDB/pull/2/commits/f89e35c1028c3d8ca7985378c969a3f7430cd26e)

File: README.md [hunk 2]
```diff
@@ -1,3 +1,44 @@
...
+Type this command
+
+```bash
+cd server
+npm run local
+```
+
+### Execute query
+
+We are able to install [GraphQL Playground](https://github.com/prisma-labs/graphql-playground) for execute query.
```
Comment (chompy18) [inline README.md]: There is no need for the install. Running the server and pointing the browser to localhost:4000 will bring it up. It is part of Apollo.

File: README.md [hunk 3]
```diff
@@ -1,3 +1,44 @@
...
+
+## How to use
+
+### Execute loal server
+
+Two ways for execute local server
+
+#### Using [Visual Studio Code](https://azure.microsoft.com/en-us/products/visual-studio-code/)
+
+Modify configurations of `launch.json`.
```
Comment (chompy18) [inline README.md]: Vscode debug config is already in place, no need for any modification.

**Review by chompy18 — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (ufoo68) [reply README.md]: Sorry, this is my problem of PATH settings. Fixed in [67d2e2f](https://github.com/jsBlackBelt/TheSportsDB/pull/2/commits/67d2e2f5506ea82bd137cb35bdd5e9f008303969)

File: README.md (hunk 2 above)
Comment (ufoo68) [reply README.md]: I didn't know there was such a function. Fixed in [67d2e2f](https://github.com/jsBlackBelt/TheSportsDB/pull/2/commits/67d2e2f5506ea82bd137cb35bdd5e9f008303969)

**Review by sefininio — APPROVED** [review]

**Pull request merged**